src/main/java/com/dev/cinema/model/dto/MovieSessionDtoRequest.java

```diff
@@ -1,8 +1,11 @@
 package com.dev.cinema.model.dto;
 
+import jakarta.validation.constraints.NotNull;
+
 public class MovieSessionDtoRequest {
     private Long movieId;
     private Long cinemaHallId;
```

Comment:
Can we create movie session without movie or cinema hall?

Reply:
Changed.